src/test/java/org/springframework/data/couchbase/util/UnmanagedTestCluster.java

```diff
@@ -59,7 +59,7 @@ ClusterType type() {
 
 	@Override
 	TestClusterConfig _start() throws Exception {
-		bucketname = UUID.randomUUID().toString();
+		bucketname = "my_bucket"; // UUID.randomUUID().toString();
```

Comment:
why is this hardcoded now?

Reply:
With the creation of multiple scopes and collections and the indexing, the initialization is getting long.  This now reuses the bucket, scopes, collections etc. if they already exist.